apps/client/src/intros/DakkapelIntro.js

```diff
@@ -12,14 +13,12 @@ export default () => (
     <List variant="bullet">
       <ListItem>
         Wilt u de dakkapel plaatsen op een woonwagen, een tijdelijk gebouw, een
-        blokhut of een vakantiehuis? Bel dan de gemeente op{" "}
-        <a href="tel:14020">14 020</a>, maandag tot en met vrijdag van 08.00 uur
-        tot 18.00 uur
+        blokhut of een vakantiehuis?{" "}
+        <ContactSentence eventName="Intro - Extraordinary situation" />
```

Comment:
This we will change in a separate PR @svenjens

Reply:
👍